core-rust/core-api-server/src/core_api/conversions/substates/royalty_module.rs

```diff
@@ -7,17 +7,14 @@ use radix_engine_queries::typed_substate_layout::*;
 
 pub fn to_api_component_royalty_substate(
     context: &MappingContext,
-    substate: &FieldSubstate<ComponentRoyaltySubstate>,
+    substate: &ComponentRoyaltyAccumulatorFieldSubstate,
 ) -> Result<models::Substate, MappingError> {
-    Ok(field_substate!(
+    Ok(field_substate_versioned!(
         substate,
         RoyaltyModuleFieldState,
-        ComponentRoyaltySubstate {
-            enabled,
-            royalty_vault,
-        },
+        ComponentRoyaltySubstate { royalty_vault },
         Value {
-            is_enabled: *enabled,
+            is_enabled: true, // TODO(during review): drop it instead?
```

Comment:
I don't mind either way. The substate won't exist any more if royalties aren't enabled